core-services/ci-secret-bootstrap/README.md

```diff
@@ -1,4 +1,5 @@
-# ci-secret-bootstrap
```

Comment:
Why the removal of markdown formatting?

Reply:
I didn't removed, I changed it to `=========`.